src/components/Buttons.js

```diff
@@ -260,74 +260,81 @@ export class ItemMenuButton extends Component {
   }
 }
 
+export const preventOverflowPopperConfig = {
+  modifiers: {
+    preventOverflow: {
+      boundariesElement: 'window'
+    }
+  }
+};
+
 export class EditPencilForTableRow extends Component {
   render() {
     return (
-      <OverlayTrigger
-        key='bottom'
-        placement='bottom'
-        overlay={
-          <Tooltip id='edit-row'>
-            {translate('common.edit')}
-          </Tooltip>
-        }
-      >
-        <td className='actions'>
-          <p onClick={this.props.clickAction}>
+      <td className='actions'>
+        <p onClick={this.props.clickAction}>
+          <OverlayTrigger
+            placement='bottom'
+            overlay={
+              <Tooltip id='edit-row'>
+                {translate('common.edit')}
+              </Tooltip>
+            }
+          >
```

Comment:
you might want to add popperConfig={preventOverflowPopperConfig} here

Reply:
:laughing: How did I miss that? Good catch!